### PR TITLE
Release automation

### PR DESCRIPTION
**Pull request by jabrah**
* Remove old Docker dev environment
* Add initial GH workflow for (manual) release

### CODE REVIEW

**Comment by jabrah** [comment]
Ah, I see that including the updating of pass-ui:latest Docker image in the CI workflow results in an entry marked as skipped on all PRs: 
image

Maybe we should move it into a separate workflow, then, in order to not have that show up on PRs

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,14 +33,48 @@ jobs:
...
-          node-version: 18.x
+          node-version: ${{ matrix.node-version }}
           cache: yarn
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
       - name: Run Tests
         run: yarn test
+
+  # Only run after test job for pushes to `main`
+  update-latest:
```
Comment (jaredgalanis) [inline .github/workflows/ci.yml:54]: we talked about moving this to a separate workflow

**Review by jaredgalanis — APPROVED** [review]
Good to go once the changes we discussed are in

**Comment by jabrah** [comment]
Updating Docker image in separate workflow will run on all merges to `main` now

**Pull request merged**